.github/workflows/test.yaml

```diff
@@ -97,11 +114,12 @@ jobs:
       - name: Load Docker images.
         uses: ./
         with:
-          key: docker-cache-test-${{ github.run_id }}-${{ github.run_attempt }}
+          key: docker-cache-test-${{ matrix.os }}-${{github.run_id }}-${{ github.run_attempt }}
       - name: Verify Docker loaded empty image from cache.
         run: |
           description="$(docker inspect --format '{{ index .Config.Labels "description" }}' empty)"
           [[ $description == 'empty image' ]]
+        shell: bash
       - name: Delete test cache if permitted (i.e., workflow not triggered from fork).
         if: always()
         run: >
```

Comment:
I overlooked the fact that we forgot to update the key in the delete cache step.

Reply:
Oooh thank you for catching this! I'll try to fix it now!